aiida/backends/managers/profile_access.py

```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
```

Comment:
Shouldn't this file also not just belong in `aiida.manage.configuration`? It is there where config and profile loading/access is organized and where this is also being called. I think @chrisjsewell was getting rid of this entire module soon anyway.

Reply:
I moved it to `aiida.manage.profile_access`, it seemed more fitting that `configuration`, but let me know if you disagree.

Reply:
I like it :+1:

Reply:
Sounds good to me.

Random note that, as it does pertain to this line, but https://www.python.org/dev/peps/pep-3120/ makes the default encoding for python source UTF-8, so these `# -*- coding: utf-8 -*-` lines are kinda a legacy thing